mathics/core/formatter.py

```diff
@@ -113,7 +113,15 @@ def boxes_to_format(boxes, format, **options) -> str:  # Maybe Union[str, bytear
     Translates a box structure ``boxes`` to a file format ``format``.
 
     """
-    return lookup_method(boxes, format)(boxes, **options)
+    try:
+        return lookup_method(boxes, format)(boxes, **options)
+    except RecursionError:
+        # This is just for debugging.
```

Comment:
Is this supposed to be kept?

Reply:
For some reason, in mathics-Django, I got this exception. However, I was not able to determine why it was raised, and the traceback crashes my browser. Do you have a better idea to have the diagnostic?

Reply:
Maybe a print?